Sources/ATTNParameterValidation.m

```diff
@@ -36,6 +44,18 @@ + (void)verifyStringOrNil:(nullable NSString *)inputValue inputName:(nonnull con
   }
 }
 
++ (void)verifyString:(nullable NSString *)inputValue inputName:(nonnull const NSString *)inputName {
+  if ([ATTNParameterValidation isNil:inputValue]) {
+    return;
+  }
+  if (![ATTNParameterValidation isString:inputValue]) {
+    [NSException raise:@"Bad Identifier" format:@"%@ should be NSString", inputName];
+  }
```

Comment:
I think we shouldn't be using `NSException raise` because that can crash clients' apps - let's just log an error here instead